File: 14-longest-common-prefix/14-longest-common-prefix.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        if (strs.length == 0) {return "";}
        String ans = strs[0];
        for (int i = 0; i < strs.length; i++) {
            while (strs[i].indexOf(ans) != 0) {
                ans = ans.substring(0, ans.length()-1);
                if (ans.isEmpty()) {return "";}
            }
        }
        return ans;
    }
}

File: 14-longest-common-prefix/14-longest-common-prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        answer = ""
        for i in zip(*strs):
            if len(set(i)) == 1:
                answer += i[0]
            else:
                break
        return answer